developer-guides/rest-api/README.md

```diff
@@ -4,7 +4,7 @@ The REST API allows you to control and extend Rocket.Chat with ease.
 
 > **This API is a work in progress, so feel free to test, ask us questions, and submit Pull Requests!**
 
-If you are an end-user and not a dev or a tester, [create an issue](https://github.com/RocketChat/Rocket.Chat/issues/new) to request new APIs -- and consider [making a donation](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ZL94ZE6LGVUSN) to the project.
+If you are an end-user and not a dev or a tester, [create a NFR](https://forums.rocket.chat/c/feature-requests) to request new APIs -- and consider [making a donation](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ZL94ZE6LGVUSN) to the project.
```

Comment:
I think we can have `create a new feature request` instead of using `NFR` here.

Reply:
Agreed, changed this.